src/block-components/image/image-optimizer-polyfill.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * WordPress dependencies
+ */
+import domReady from '@wordpress/dom-ready'
+
+class ImageOptimizerPolyfill {
```

Comment:
Please add a description of what this file is for, what it does, why we need it, and when it is loaded